airbyte-integrations/connectors/source-aha/source_aha/source.py

```diff
@@ -13,6 +21,49 @@
 
 
 # Declarative Source
-class SourceAha(YamlDeclarativeSource):
+class Aha(YamlDeclarativeSource):
     def __init__(self):
         super().__init__(**{"path_to_yaml": "manifest.yaml"})
+
+
+def check_existence(config: Mapping[str, Any], url_param: str) -> Union[tuple[bool, None], tuple[bool, str]]:
+    response = requests.head(urljoin(config['url'], f"/api/v1/{url_param}"), headers={"Authorization": f"Bearer {config['api_key']}"})
+    if response.status_code == 200:
+        return True, None
+    return False, response.text
+
+
+class AhaEmptyBase(Stream):
+    primary_key = None
+
+    def read_records(self, sync_mode: SyncMode, cursor_field: List[str] = None, stream_slice: Mapping[str, Any] = None,
+                     stream_state: Mapping[str, Any] = None) -> Iterable[StreamData]:
+        return []
+
+
+class Goals(AhaEmptyBase):
+    pass
+
+
+class Features(AhaEmptyBase):
+    pass
```

Comment:
I don't believe the change you're doing it is correct.

Reply:
> I don't believe the change you're doing it is correct.

Can you evaluate that, please?

Reply:
The aha.io API docs https://www.aha.io/api/resources/features/list_features

Are you trying to develop something with this connector? If so, only remove from the configured catalog and it should work fine for your case.